integration-tests/project-folder/sources/meta-fixtures/diagnostics.bb

```diff
@@ -0,0 +1,3 @@
+python () {
```

Comment:
This layer won't parse. I suggest renaming meta-fixtures to meta-errors-2.
That way we could use a meta-fixtures in the future with working recipes.
Another possibility that may be better would be to move to meta-errors and use BBMASK to mask other erroneous recipes we are not interested in for the current test.

Reply:
We have some functionalities that require the files to be placed into a proper Bitbake project, and some others that do not. When it is not required, it is simpler not to build the recipe needlessly.

The diagnostics are special in the way that they are a bit of both: The parsing requires a proper Bitbake project, but embedded languages do not.

I spent a lot of time trying to include (temporarily) my integration test into `bitbake-parse.test.ts`. For unknown reasons, the "embedded languages" diagnostics never appear, no matters how long I wait. This is the reason why they are tested separately for now, without building the recipe.

Reply:
There is an example for temporarily including a layer that doesn't parse in https://github.com/yoctoproject/vscode-bitbake/blob/29f5e2d81a744db90e2adacaa284a34aeb378cba/integration-tests/src/tests/bitbake-parse.test.ts#L75

Reply:
Yes. I've spent a lot of time trying include the same test (and variations of it) into this test, and for unknown reasons the embedded languages' diagnostics never appear for error.bb (but they do appear for the "embedded languages files")

Reply:
Make sure you appropriately wait for the related background scanning tasks to run at the right time. All the waiting done in the test suite and before hooks matter.

Reply:
yes, the problem is not the parsing. The diagnostics from the bitbake parsing still appeared properly in the tests I made. The problem is with the diagnostics for the embedded languages